luminoth/tools/server/web.py

```diff
@@ -22,84 +16,6 @@ def get_image():
     return image
 
 
```

Comment:
Add LOADED_MODELS/cached graph to web, or else there is a huge performance drop.